### PR TITLE
Homework 2. Components.

### PR DESCRIPTION
**Pull request by dreemius**

### CODE REVIEW

File: client/src/app/app.routes.ts [hunk 1]
```diff
@@ -5,9 +5,7 @@ import { StatisticsComponent } from "./statistics/statistics.component";
 import { TaskComponent } from "./task/task.component";
 import { TaskEditComponent } from "./task/edit/task-edit.component";
 
-/** this is routes block */
-
-// 11111
+/** this is routes block, so lets commit */
 export const routes: Routes = [
```
Comment (fullstackdb) [inline client/src/app/app.routes.ts]: all seems good
nice work

**Review by fullstackdb — CHANGES_REQUESTED** [review]
need some changes

File: client/src/app/app.routes.ts (hunk 1 above)
Comment (dreemius) [reply client/src/app/app.routes.ts]: Fixed

**Review by fullstackdb — APPROVED** [review]
seems as done

**Pull request merged**